proto/ibc/applications/middleware/fee/v1/tx.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+
+// Msg defines the ibc/fee Msg service.
+service Msg {
+  // RegisterCounterPartyAddress defines a rpc handler method for MsgRegisterCounterPartyAddress
+  // RegisterCounterpartyAddress is called by the relayer on each channelEnd and allows them to specify their
+  // counterparty address before relaying This ensures they will be properly compensated for forward relaying since
+  // destination chain must send back relayer's source address (counterparty address) in acknowledgement This function
+  // may be called more than once by a relayer, in which case, latest counterparty address is always used.
+  rpc RegisterCounterPartyAddress(MsgRegisterCounterPartyAddress) returns (MsgRegisterCounterPartyAddressResponse);
+  // EscrowPacketFee defines a rpc handler method for MsgEscrowPacketFee
+  // EscrowPacketFee is an open callback that may be called by any module/user that wishes to escrow funds in order to
+  // incentivize the relaying of the given packet.
+  rpc EscrowPacketFee(MsgEscrowPacketFee) returns (MsgEscrowPacketFeeResponse);
+  // PayFee defines a rpc handler method for MsgPayFee
+  // PayFee is a callback implemented by fee module called by the ICS-4 AcknowledgePacket handler.
+  rpc PayFee(MsgPayFee) returns (MsgPayFeeResponse);
+  // PayTimeoutFee defines a rpc handler method for MsgPayTimeoutFee
+  // PayFee is a callback implemented by fee module called by the ICS-4 TimeoutPacket handler.
+  rpc PayTimeoutFee(MsgPayTimeoutFee) returns (MsgPayTimeoutFeeResponse);
+}
+
+// MsgRegisterCounterPartyAddress is the request type for registering the counter party address
+message MsgRegisterCounterPartyAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string address              = 1 [(gogoproto.moretags) = "yaml:\"address\""];
+  string counterparty_address = 2 [(gogoproto.moretags) = "yaml:\"address\""];
+}
+
+// MsgRegisterCounterPartyAddressResponse defines the Msg/RegisterCounteryPartyAddress response type
+message MsgRegisterCounterPartyAddressResponse {}
```

Comment:
Not clear on what we want to be returning here. wdyt @AdityaSripal @colin-axner

Reply:
Just leave it blank. This is what the other msg servers do

Reply:
`CounterParty` -> `Counterparty`